apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -186,164 +193,195 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
           })}
           href={paths.skillsAndExperiences(user.id)}
         >
-          <HeroCardExperienceItem
-            sectionName={intl.formatMessage({
-              defaultMessage: "Work experience",
-              id: "giUfys",
+          <StatusItem
+            title={intl.formatMessage({
+              defaultMessage: "Work experiences",
+              id: "LOmX3T",
               description: "Title for work experience section",
             })}
             itemCount={workExperiences?.length}
             icon={BriefcaseIcon}
-            color="primary"
           />
-          <HeroCardExperienceItem
-            sectionName={intl.formatMessage({
-              defaultMessage: "Education experience",
-              id: "u6LIbY",
+          <StatusItem
+            title={intl.formatMessage({
+              defaultMessage: "Education and certificates",
+              id: "PFoM2I",
               description: "Title for education experience section",
             })}
             itemCount={educationExperiences?.length}
             icon={BookOpenIcon}
-            color="secondary"
           />
-          <HeroCardExperienceItem
-            sectionName={intl.formatMessage({
-              defaultMessage: "Volunteer and community experience",
-              id: "Rz7WtH",
+          <StatusItem
+            title={intl.formatMessage({
+              defaultMessage: "Community participation",
+              id: "Uy5Dg2",
               description: "Title for community experience section",
             })}
             itemCount={communityExperiences?.length}
             icon={UsersIcon}
-            color="tertiary"
           />
-          <HeroCardExperienceItem
-            sectionName={intl.formatMessage({
-              defaultMessage: "Personal experience",
-              id: "wTFUPE",
+          <StatusItem
+            title={intl.formatMessage({
+              defaultMessage: "Personal learning",
+              id: "UDMUHH",
               description: "Title for personal experience section",
             })}
             itemCount={personalExperiences?.length}
             icon={LightBulbIcon}
-            color="quaternary"
           />
-          <HeroCardExperienceItem
-            sectionName={intl.formatMessage({
-              defaultMessage: "Award",
-              id: "+ikQY0",
+          <StatusItem
+            title={intl.formatMessage({
+              defaultMessage: "Awards and recognition",
+              id: "mWnekb",
               description: "Title for award section",
             })}
             itemCount={awardExperiences?.length}
             icon={StarIcon}
-            color="quinary"
+          />
+          <StatusItem
+            title={intl.formatMessage({
+              defaultMessage: "Qualified recruitments",
+              id: "2dpDPq",
+              description: "Title for qualified recruitments section",
+            })}
+            itemCount={0}
```

Comment:
Is this supposed to be wired up yet?

Reply:
I think this link is supposed to go here, which doesn't exist yet.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/cec211ef-64ba-4997-9bc9-2aba6c881dae)

Reply:
Added note to subsequent issue. https://github.com/GCTC-NTGC/gc-digital-talent/issues/6773#issuecomment-1584545972